### ISSUE TITLE
NoSuchFieldError: INFORMATION_COLLECTION2 when trying to update CVE database with CLI

### ISSUE THREAD

**Issue by s3nl** [dependency-check/DependencyCheck#2767]
Please delete any un-needed section from the following issue template:

### Reporting Errors
Hi,

Ever since we upgraded to 5.3.2 we get an error when we try to update the database.

We are using this command:
dependency-check-5/dependency-check/bin/dependency-check.sh \
--project "Download vulnerability database" \
--out "./dependency-check-reports" \
--data "./dependency-check-data" \
--cveUrlBase "https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-%d.json.gz" \
--cveUrlModified "https://nvd.nist.gov/feeds/json/cve/1.0/nvdcve-1.0-modified.json.gz" \
--retireJsUrl "https://raw.githubusercontent.com/Retirejs/retire.js/master/repository/jsrepository.json" \
--proxyserver "<proxyserver>" \
--proxyport <proxyport> \
--updateonly true

When I try to update the CVE database I get the following error:
Exception in thread "main" java.util.ServiceConfigurationError: org.owasp.dependencycheck.analyzer.Analyzer: Provider org.owasp.dependencycheck.analyzer.PEAnalyzer could not be instantiated
Caused by: java.lang.NoSuchFieldError: INFORMATION_COLLECTION2

Here’s the stacktrace:
Exception in thread "main" java.util.ServiceConfigurationError: org.owasp.dependencycheck.analyzer.Analyzer: Provider org.owasp.dependencycheck.analyzer.PEAnalyzer could not be instantiated
       at java.util.ServiceLoader.fail(ServiceLoader.java:232)
       at java.util.ServiceLoader.access$100(ServiceLoader.java:185)
       at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:384)
       at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
       at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
       at org.owasp.dependencycheck.analyzer.AnalyzerService.getAnalyzers(AnalyzerService.java:103)
       at org.owasp.dependencycheck.Engine.loadAnalyzers(Engine.java:302)
       at org.owasp.dependencycheck.Engine.initializeEngine(Engine.java:270)
       at org.owasp.dependencycheck.Engine.<init>(Engine.java:259)
       at org.owasp.dependencycheck.Engine.<init>(Engine.java:234)
       at org.owasp.dependencycheck.Engine.<init>(Engine.java:224)
       at org.owasp.dependencycheck.App.runUpdateOnly(App.java:384)
       at org.owasp.dependencycheck.App.run(App.java:164)
       at org.owasp.dependencycheck.App.main(App.java:81)
Caused by: java.lang.NoSuchFieldError: INFORMATION_COLLECTION2
       at org.owasp.dependencycheck.analyzer.PEAnalyzer.<clinit>(PEAnalyzer.java:79)
       at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
       at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
       at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
       at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
       at java.lang.Class.newInstance(Class.java:442)
       at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:380)
       ... 11 more

Can anyone help shine some light as to why we’re getting this error and how to solve it?

**Comment by jeremylong** [comment]
Looks like you tried to overlay a new version of ODC over an existing install? It looks like you have a classpath issue.  Also, for update only it looks like you are adding args with old default values. If all you are doing is updating the database you could run:

```
dependency-check.sh --data "./dependency-check-data" --updateonly
```

**Comment by s3nl** [comment]
that is exactly what happened... should the old ODC be deleted, new one added before proceeding?

**Comment by jeremylong** [comment]
Yes, delete the old ODC.

**Comment by s3nl** [comment]
so I deleted the old version and used a fresh one. As the job runs I can see what's in it, and it's the same from what's in the ZIP file I donwloaded. Here's a list:
dependency-check-5/dependency-check/LICENSE.txt
14:20:53 dependency-check-5/dependency-check/NOTICE.txt
14:20:53 dependency-check-5/dependency-check/README.md
14:20:53 dependency-check-5/dependency-check/bin/
14:20:53 dependency-check-5/dependency-check/bin/dependency-check.bat
14:20:53 dependency-check-5/dependency-check/bin/dependency-check.sh
14:20:53 dependency-check-5/dependency-check/bin/duo-jenkins-script.sh
14:20:53 dependency-check-5/dependency-check/lib/
14:20:53 dependency-check-5/dependency-check/lib/annotations-19.0.0.jar
14:20:53 dependency-check-5/dependency-check/lib/ant-1.9.9.jar
14:20:53 dependency-check-5/dependency-check/lib/checker-qual-2.10.0.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-beanutils-1.9.4.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-cli-1.4.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-collections-3.2.2.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-compress-1.19.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-digester-1.8.1.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-io-2.6.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-jcs-core-2.2.1.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-lang3-3.9.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-logging-1.2.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-text-1.8.jar
14:20:53 dependency-check-5/dependency-check/lib/commons-validator-1.6.jar
14:20:53 dependency-check-5/dependency-check/lib/compiler-0.9.5.jar
14:20:53 dependency-check-5/dependency-check/lib/cpe-parser-2.0.2.jar
14:20:53 dependency-check-5/dependency-check/lib/dependency-check-cli-5.3.2.jar
14:20:53 dependency-check-5/dependency-check/lib/dependency-check-core-5.3.2.jar
14:20:53 dependency-check-5/dependency-check/lib/dependency-check-utils-5.3.2.jar
14:20:53 dependency-check-5/dependency-check/lib/error_prone_annotations-2.3.4.jar
14:20:53 dependency-check-5/dependency-check/lib/failureaccess-1.0.1.jar
14:20:53 dependency-check-5/dependency-check/lib/gson-2.8.6.jar
14:20:53 dependency-check-5/dependency-check/lib/guava-28.2-jre.jar
14:20:53 dependency-check-5/dependency-check/lib/h2-1.4.199.jar
14:20:53 dependency-check-5/dependency-check/lib/j2objc-annotations-1.3.jar
14:20:53 dependency-check-5/dependency-check/lib/jackson-annotations-2.9.10.jar
14:20:53 dependency-check-5/dependency-check/lib/jakarta.activation-1.2.1.jar
14:20:53 dependency-check-5/dependency-check/lib/javax.inject-1.jar
14:20:53 dependency-check-5/dependency-check/lib/javax.json-1.0.4.jar
14:20:53 dependency-check-5/dependency-check/lib/javax.ws.rs-api-2.0.1.jar
14:20:53 dependency-check-5/dependency-check/lib/jcl-over-slf4j-1.7.28.jar
14:20:53 dependency-check-5/dependency-check/lib/joda-time-2.10.4.jar
14:20:53 dependency-check-5/dependency-check/lib/json-20180130.jar
14:20:53 dependency-check-5/dependency-check/lib/jsoup-1.13.1.jar
14:20:53 dependency-check-5/dependency-check/lib/jsr305-3.0.2.jar
14:20:53 dependency-check-5/dependency-check/lib/listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
14:20:53 dependency-check-5/dependency-check/lib/logback-classic-1.2.3.jar
14:20:53 dependency-check-5/dependency-check/lib/logback-core-1.2.3.jar
14:20:53 dependency-check-5/dependency-check/lib/lucene-analyzers-common-8.5.0.jar
14:20:53 dependency-check-5/dependency-check/lib/lucene-core-8.5.0.jar
14:20:53 dependency-check-5/dependency-check/lib/lucene-queries-8.5.0.jar
14:20:53 dependency-check-5/dependency-check/lib/lucene-queryparser-8.5.0.jar
14:20:53 dependency-check-5/dependency-check/lib/lucene-sandbox-8.5.0.jar
14:20:53 dependency-check-5/dependency-check/lib/mailapi-1.6.5.jar
14:20:53 dependency-check-5/dependency-check/lib/minlog-1.3.jar
14:20:53 dependency-check-5/dependency-check/lib/ossindex-service-api-1.3.0.jar
14:20:53 dependency-check-5/dependency-check/lib/ossindex-service-client-1.3.0.jar
14:20:53 dependency-check-5/dependency-check/lib/package-url-java-1.1.1.jar
14:20:53 dependency-check-5/dependency-check/lib/packageurl-java-1.1.1.jar
14:20:53 dependency-check-5/dependency-check/lib/pecoff4j-0.0.2.1.jar
14:20:53 dependency-check-5/dependency-check/lib/retirejs-core-3.0.2.jar
14:20:53 dependency-check-5/dependency-check/lib/semver4j-3.1.0.jar
14:20:53 dependency-check-5/dependency-check/lib/slf4j-api-1.7.30.jar
14:20:53 dependency-check-5/dependency-check/lib/spotbugs-annotations-4.0.0.jar
14:20:53 dependency-check-5/dependency-check/lib/toml4j-0.7.2.jar
14:20:53 dependency-check-5/dependency-check/lib/velocity-engine-core-2.2.jar
14:20:53 dependency-check-5/dependency-check/licenses/
14:20:53 dependency-check-5/dependency-check/licenses/commons-cli/
14:20:53 dependency-check-5/dependency-check/licenses/commons-cli/LICENSE.txt

Is it maybe remembring something else from the old version? we upgraded from 5.3 to 5.3.2.
